pkg/kube/resources.go

```diff
@@ -91,12 +94,14 @@ func CreateResourceProviderFromPath(directory string) (*ResourceProvider, error)
 		}
 		contents, err := ioutil.ReadFile(path)
 		if err != nil {
+			logrus.Errorf("Error reading file %v", path)
 			return err
 		}
 		specs := regexp.MustCompile("\n-+\n").Split(string(contents), -1)
 		for _, spec := range specs {
 			err = addYaml(spec)
 			if err != nil {
+				logrus.Errorf("Error parsing YAML %v", err)
```

Comment:
FYI this error will be logged twice (once in `addYaml`), but maybe for the best

Reply:
Yeah, that was intentional. I wanted to have more specific logging in `addYaml` but also add this as a failsafe if an error hadn't already gotten logged there. It will likely be repetitive if they run into an error, but does ensure we catch things added to `addYaml` in the future that may not get the same kind of logging in the future. I don't know, that could be overkill.

Reply:
👍 sounds good to me